front-project/src/index.js

```diff
@@ -16,6 +18,14 @@ const router = createBrowserRouter([
     path: "auth",
     element: <AuthPage />,
   },
+  {
```

Comment:
Add pages SignInPage and SignUpPage and render forms inside these pages (later default layout with header/footer/sidebar will be added in addition to form components)